viz_scripts/bin/generate_plots.py

```diff
@@ -43,10 +44,18 @@
 # Check if the dynamic config contains dynamic labels 'label_options'
 # Passing the boolean flag is not enough, bcos we need to trace through the dynamic_config - pass label_options URL.
 if 'label_options' in dynamic_config:
-    has_dynamic_labels = True
     dynamic_labels_url = dynamic_config['label_options']
+
+# Parse through the dynamic_labels_url: Create a new function for this
+if  dynamic_labels_url:
+    req = requests.get(dynamic_labels_url)
+    if req.status_code != 200:
+        print("Unable to download dynamic_labels, status code: {r.status_code}")
```

Comment:
nit: (**future**): why is this `if` outside the `if 'label_options' in dynamic_config:`
It seems like if we have label options, we must have `dynamic_labels_url` and can put all this into that conditional

Reply:
Moved the `if dynamic_labels_url:` inside `if 'label_options' in dynamic_config:`.

Reply:
@iantei this was marked as a *future* fix, I do not expect it to be fixed right now. Now that you have fixed it, we can include it, but we can also avoid unnecessary code churn while getting to the point of finalizing features.

Reply:
Okay